src/components/smart/DeploymentDetailsPage/DeploymentDetailsPage.tsx

```diff
@@ -250,14 +250,52 @@ class DeploymentDetailsPage extends React.Component<Props, State> {
       },
     );
 
-    instanceStore.loadInstancesDetails({
-      endpointId: details.origin_endpoint_id,
-      instances: details.instances.map(n => ({ id: n })),
-      cache: options.cache,
-      quietError: false,
-      env: details.source_environment,
-      targetProvider: targetEndpoint?.type,
+    const isDeploymentCompleted = this.isDeploymentCompleted();
+    const hasStoredVmInfo =
+      details.info && Object.keys(details.info).length > 0;
+
+    if (isDeploymentCompleted && hasStoredVmInfo) {
```

Comment:
If we already have stored VM info, there's no point in checking whether the deployment's completed or not. Same goes for transfer. Even though it has no fully completed executions, all it takes to save export_info is a completed `GET_INSTANCE_INFO`. Although, I'd more precisely check for `details.info.export_info` as well.